packages/esm-patient-chart-app/src/ui-components/patient-chart.component.tsx

```diff
@@ -19,10 +20,11 @@ const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({ match
   const { patientUuid, view, subview } = match.params;
   const [loading, patient] = useCurrentPatient(patientUuid);
   const state = useMemo(() => ({ patient, patientUuid }), [patient, patientUuid]);
+  const { active } = useWorkspace();
 
   const mainClassName = `
     omrs-main-content 
-    ${styles.chartContainer} 
+    ${styles.chartContainer} ${active && styles.marginRight}
```

Comment:
Not sure if we need this. It should be automatically layouted well. Flexbox to the rescue!! 🦸